src/debugger/scriptImporter.ts

```diff
@@ -86,4 +87,14 @@ export class ScriptImporter {
                 return new FileSystem().writeFile(sourceMappingLocalPath, updatedContent);
             });
     }
+
+    /**
+     * Changes the port of the url to be the one configured as this.packagerPort
+     */
+    private overridePackagerPort(urlToOverride: string): string {
```

Comment:
The argument still holds, even though it is not used in the case of the multiple instance scenario. The `ScriptImporter` should not care if we use port forwarding or not.